### ISSUE TITLE
Tests with different parameters

### ISSUE THREAD

**Issue by jcribeiro98** [Gradiant/ai-project-cifar10#9]
## Description

Studying how changes in the parameters of the model affects both the accuracy and the f1 of the model

## Assumptions

Due to the high amount of variability that the loss function presents during training (check: http://10.10.30.58:8999/#/metric/train%2Floss?runs=[%228e12317985294775bb2b4ad5d87fe553%22]&experiment=64&plot_metric_keys=[%22train/loss%22]&plot_layout={%22autosize%22:true,%22xaxis%22:{},%22yaxis%22:{}}&x_axis=relative&y_axis_scale=linear&line_smoothness=1&show_point=false&deselected_curves=[]&last_linear_y_axis_range=[]), it would be reasonable to assume that reducing the learning rate could reduce the variability of the loss function, and hence, produce a more accurate model.

**Comment by jcribeiro98** [comment]
## Results

Reproducing the experiment multiple times with different parameters we found the following results

| Epochs 	| Learning Rate 	| Loss 	| f1 score 	| Accuracy: top 1 	| Accuracy: top 5 	|
|---	|---	|---	|---	|---	|---	|
| 8 	| 1e-5 	| 0.569 	| 0.76238 	| 76.32 	| 98.43 	|
| 8 	| 1e-6 	| 0.165 	| 0.84285 	| 84.38 	| 99.41 	|
| 12 	| 1e-6 	| 0.083 	| 0.84388 	| 84.43 	| 99.05 	|


Indeed, reducing the learning rate helped with the variability of the loss function during training, which later translates to an increase in accuracy. (check http://10.10.30.58:8999/#/metric/train%2Floss?runs=[%228f453a7e147043cd8558eb29a3938dfd%22]&experiment=64&plot_metric_keys=[%22train/loss%22]&plot_layout={%22autosize%22:true,%22xaxis%22:{},%22yaxis%22:{}}&x_axis=relative&y_axis_scale=linear&line_smoothness=1&show_point=false&deselected_curves=[]&last_linear_y_axis_range=[])

Furthermore, we've found that increasing the total number of epochs reduces the loss (which is to be expected) but doesn't make significant changes on the accuracy nor the F1 score.
